regression/tests/studio/test_course_pages.py

```diff
@@ -148,3 +149,97 @@ def test_view_live_pages(self):
         courseware_page = CoursewarePage(
             self.browser, get_course_info())
         courseware_page.wait_for_page()
+
+    def test_drag_and_drop_of_pages(self):
+        """
+        Verifies that user can drag and drop pages and they appear in the
+        proper order in LMS
+        """
+
+        # Get all pages which are default and moveable
+        moveable_pages = self.pages_page.get_moveable_default_pages()
+        # Get current order of pages
+        moveable_pages_order =\
+            self.pages_page.get_movable_default_pages_order()
+
+        # Drag the last page and drop it on the second last page.
+        self.pages_page.drag_and_drop(
+            moveable_pages[-1],
+            moveable_pages[-2]
+        )
+
+        temp_page = moveable_pages_order[-2]
+        moveable_pages_order[-2] = moveable_pages_order[-1]
+        moveable_pages_order[-1] = temp_page
+
+        # Asset that order of pages has been changed.
+        self.assertEqual(
+            self.pages_page.get_movable_default_pages_order(),
+            moveable_pages_order
+        )
+
+        course_info = get_course_info()
+        course_page = CourseInfoPageExtended(
+            self.browser, get_course_key(course_info)
+        )
+        course_page.visit()
+        pages_in_tab = course_page.get_pages_name()
+
+        # Convert lists pages_in_tab and moveable_pages_order to string.
+        # It makes it easier to assert the order.
+        string_of_tab_pages = ''.join(pages_in_tab)
```

Comment:
Ah, I guess we need to refactor it a bit. Specifically, to use list assertion, I am considering refining the list returned by get_pages_name.